R/pkg/R/DataFrame.R

```diff
@@ -1437,6 +1437,29 @@ dapplyInternal <- function(x, func, schema) {
     schema <- structType(schema)
   }
 
+  arrowEnabled <- sparkR.conf("spark.sql.execution.arrow.enabled")[[1]] == "true"
```

Comment:
@felixcheung, I will deduplicate this checker across all R optimization after this PR if that's fine to you too. I was trying to work on multiple features so wanted to avoid conflicts.

Reply:
I filed a JIRA at https://issues.apache.org/jira/browse/SPARK-26920